copulas/univariate/KDEUnivariate.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import numpy as np
+
+from copulas.univariate.UnivariateDistrib import UnivariateDistrib
+from scipy.stats import gaussian_kde
+import scipy.optimize as optimize
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KDEUnivariate(UnivariateDistrib):
+    """ Use Kernel Density Estimation to infer univariate distribution """
+
+    def __init__(self):
+        super(KDEUnivariate, self).__init__()
+        self.column = None
+        self.param = None
+
+    def fit(self, column):
+        self.column = column
```

Comment:
What is `self.column` used for? I see no any other reference to it anywhere in the class.

Reply:
It is not used in this class. But it might be used in other classes and it is easier to retrieve the data.

Reply:
This may not be the best of reasons:
- If another class is calling this one, they will have access to columns, as they'll have passed it on the call to `fit`.
- As a general rule classes shouldn't have attributes that they are no expecting to use. There are expections, but usually is this way